src/main/java/org/dbflute/intro/app/web/dfprop/schemapolicy/statement/DfpropSchemapolicyStatementAction.java

```diff
@@ -65,8 +66,39 @@ private SchemaPolicyStatement mappingToStatement(DfpropRegisterSchemaPolicyState
     //                       GetschemapolicyStatementSubject
     //                       -------------------------------
     @Execute
-    public JsonResponse<List<String>> subject() {
-        return asJson(dfpropInfoLogic.getStatementSubjectList());
+    public JsonResponse<List<String>> subject(DfpropSchemapolicyStatementSubjectForm form) {
+        validate(form, message -> {});
+        if (MapType.codeOf(form.mapType) == MapType.TABLE) {
+            return asJson(dfpropInfoLogic.getStatementTableMapSubjectList());
+        } else if (MapType.codeOf(form.mapType) == MapType.COLUMN) {
+            return asJson(dfpropInfoLogic.getStatementColumnMapSubjectList());
+        } else {
+            throw new NotExsistMapTypeException("存在しないMapTypeです。MapType: " + form.mapType);
+        }
+    }
+
+    public enum MapType {
```

Comment:
The class name is too ambiguous because MapType should have "wholeMap".
How about e.g. SubjectableMapType?